css/basscss.css

```diff
@@ -162,58 +137,170 @@ pre {
   overflow-x: scroll;
 }
 
+h1 { font-size: 2rem }
+h2 { font-size: 1.5rem }
+h3 { font-size: 1.25rem }
+h4 { font-size: 1rem }
+h5 { font-size: .875rem }
+h6 { font-size: .75rem }
+
+body {
+  color: #111;
+  background-color: white;
```

Comment:
I prefer the readability of color keywords, but `black` is #000, right? I think most of the color values are coming from clrs.